packages/contracts-bedrock/src/L2/interfaces/ISuperchainWETH.sol

```diff
@@ -39,8 +39,6 @@ interface ISuperchainWETH {
     /// @param _dst     Address to relay tokens to.
     /// @param _wad     Amount of tokens to relay.
     function relayERC20(address _from, address _dst, uint256 _wad) external;
-
-    function __constructor__() external;
```

Comment:
what is this change?

Reply:
This is smth that was added in `develop` by the ones doing the OP contracts manager, it was added to many interfaces

This interface was not being used before so now we have to delete that function so that we dont have to implement it